src/sinks/amqp/config.rs

```diff
@@ -8,12 +8,40 @@ use crate::{
 };
 use codecs::TextSerializerConfig;
 use futures::FutureExt;
+use lapin::{types::ShortString, BasicProperties};
 use std::sync::Arc;
 use vector_config::configurable_component;
 use vector_core::config::AcknowledgementsConfig;
 
 use super::sink::AmqpSink;
 
+/// AMQP Properties options.
+#[configurable_component]
+#[configurable(title = "Configure AMQP properties for the messages.")]
```

Comment:
```suggestion
#[configurable(title = "Configure the AMQP message properties.")]
```

? To be consistent with the other grammar.